jscomp/ml/location.ml

```diff
@@ -153,30 +227,22 @@ let print_compact ppf loc =
   end
 ;;
 
-let print_error ppf loc =
-  fprintf ppf "%a%t:" print loc print_error_prefix;
+let print_error intro ppf loc =
+  fprintf ppf "%a%t:" (print ~message_kind:`error intro) loc print_error_prefix;
 ;;
```

Comment:
Yes, here is the type change. I think we are going to use Super_location.print for print_error. Or keep the old print_error for lexer.mll?